src/Status.php

```diff
@@ -19,6 +19,9 @@ class Status
     /** @var string */
     protected $message = '';
 
+    /** @var array */
+    protected $context = [];
```

Comment:
This stops this error https://github.com/ans-group/laravel-health-check/actions/runs/4192996084/jobs/7269303493 caused by the dynamic property